assembly/src/ast/mod.rs

```diff
@@ -365,6 +407,12 @@ impl Serializable for ModuleAst {
         // asserts below are OK because we enforce limits on the number of procedure and length of
         // module docs in the module parser
 
+        assert!(self.imports.len() <= MAX_LOCAL_PROCS, "too many imports");
+        target.write_u16(self.imports.len() as u16);
+        // We don't need to serialized the keys, since they are the last name in the path
+        let values: Vec<LibraryPath> = self.imports.clone().into_values().collect();
+        values.write_into(target);
```

Comment:
Similar nit here as above: let's use a simple loop and avoid cloning.

Reply:
Does changing this to a simple loop cause some kind of issues?

Reply:
> Does changing this to a simple loop cause some kind of issues?

I don't think so. With a simple loop we are using `LibraryPath::write_into`, which writes the length of the path, and so when deserializing we know how much to read to in order to restore the LibraryPath.